packages/clerk-js/src/core/resources/Base.ts

```diff
@@ -187,4 +191,9 @@ export abstract class BaseResource {
     const { rotatingTokenNonce } = params || {};
     return this._baseGet({ forceUpdateClient: true, rotatingTokenNonce });
   }
+
+  private static shouldRethrowOfflineNetworkErrors(): boolean {
+    const experimental = BaseResource.clerk?.__internal_getOption?.('experimental');
+    return experimental?.rethrowOfflineNetworkErrors || false;
```

Comment:
If we think that this should be the new default in a new major, let's add a `TODO` comment so that we don't forget 👍